src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsRequest.js

```diff
@@ -0,0 +1,137 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import {hide as hideSidebar} from '../../../../libs/actions/Sidebar';
+import withWindowDimensions, {
+    windowDimensionsPropTypes,
+} from '../../../../components/withWindowDimensions';
+import {getNewChatOptions} from '../../../../libs/OptionsListUtils';
+
+import OptionsSelector from '../../../../components/OptionsSelector';
+import ONYXKEYS from '../../../../ONYXKEYS';
+
+const personalDetailsPropTypes = PropTypes.shape({
+    // The login of the person (either email or phone number)
+    login: PropTypes.string.isRequired,
+
+    // The URL of the person's avatar (there should already be a default avatar if
+    // the person doesn't have their own avatar uploaded yet)
+    avatar: PropTypes.string.isRequired,
+
+    // This is either the user's full name, or their login if full name is an empty string
+    displayName: PropTypes.string.isRequired,
+});
+
+const propTypes = {
+    // Callback to inform parent modal of success
+    onStepComplete: PropTypes.func.isRequired,
+
+
+    // All of the personal details for everyone
+    personalDetails: PropTypes.objectOf(personalDetailsPropTypes).isRequired,
+
+    // All reports shared with the user
+    reports: PropTypes.shape({
```

Comment:
Should this be `PropTypes.arrayOf(PropTypes.shape({...}))` ?

Reply:
@roryabraham The prop (_props.reports_), in this case, is an object with the given shape, I think. The `getNewGroupOptions` function maps it into an array.